File: 프로그래머스/Level2/점프와 순간 이동.py
# https://programmers.co.kr/learn/courses/30/lessons/12980?language=javascript

def solution(n):
    ans = 0
    while n != 0:
        if n % 2 == 0:
            n //= 2
        else:
            n -= 1
            ans += 1
    return ans

File: 프로그래머스/Level2/점프와 순간 이동.js
// https://programmers.co.kr/learn/courses/30/lessons/12980?language=javascript

const solution = (n) => {
	let answer = 0;
	while (n !== 0) {
		if (n % 2 === 0) n /= 2;
		else {
			n--;
			answer++;
		}
	}
	return answer;
};
